lightning/src/ln/channelmanager.rs

```diff
@@ -607,12 +607,22 @@ impl_writeable_tlv_based_enum!(RAAMonitorUpdateBlockingAction,
 
 /// State we hold per-peer.
 pub(super) struct PeerState<Signer: ChannelSigner> {
-	/// `temporary_channel_id` or `channel_id` -> `channel`.
+	/// `channel_id` -> `Channel`.
 	///
-	/// Holds all channels where the peer is the counterparty. Once a channel has been assigned a
-	/// `channel_id`, the `temporary_channel_id` key in the map is updated and is replaced by the
-	/// `channel_id`.
+	/// Holds all funded channels where the peer is the counterparty.
 	pub(super) channel_by_id: HashMap<[u8; 32], Channel<Signer>>,
+	/// `temporary_channel_id` -> `OutboundV1Channel`.
+	///
+	/// Holds all outbound V1 channels where the peer is the counterparty. Once an outbound channel has
+	/// been assigned a `channel_id`, the entry in this map is removed and one is created in
+	/// `channel_by_id`.
+	pub(super) outbound_v1_channel_by_id: HashMap<[u8; 32], OutboundV1Channel<Signer>>,
+	/// `temporary_channel_id` -> `InboundV1Channel`.
+	///
+	/// Holds all inbound V1 channels where the peer is the counterparty. Once an inbound channel has
+	/// been assigned a `channel_id`, the entry in this map is removed and one is created in
+	/// `channel_by_id`.
+	pub(super) inbound_v1_channel_by_id: HashMap<[u8; 32], InboundV1Channel<Signer>>,
```

Comment:
It seems we don't persist these now, which I guess is the case because `is_funding_initiated` will always be false. But this may not be the case for future maps though, correct?

Reply:
Yes. Only channels we actually get a monitor for.